src/docs/user/import.txt

```diff
@@ -838,20 +850,27 @@ $ sqoop import --connect jdbc:mysql://db.foo.com/corp \
 ----
 
 Enabling logical types in avro import and also turning on padding with 0s:
-
 ----
 $ sqoop import -Dsqoop.avro.decimal_padding.enable=true -Dsqoop.avro.logical_types.decimal.enable=true
-    --connect $CON --username $USER --password $PASS --query "select * from table_name where \$CONDITIONS"
+    --connect $MYCONN --username $MYUSER --password $MYPASS --query "select * from table_name where \$CONDITIONS"
     --target-dir hdfs://nameservice1//etl/target_path --as-avrodatafile --verbose -m 1
 
 ----
 
 Enabling logical types in avro import and also turning on padding with 0s, while specifying default precision and scale as well:
-
 ----
 $ sqoop import -Dsqoop.avro.decimal_padding.enable=true -Dsqoop.avro.logical_types.decimal.enable=true
     -Dsqoop.avro.logical_types.decimal.default.precision=38 -Dsqoop.avro.logical_types.decimal.default.scale=10
-    --connect $CON --username $USER --password $PASS --query "select * from table_name where \$CONDITIONS"
+    --connect $MYCONN --username $MYUSER --password $MYPASS --query "select * from table_name where \$CONDITIONS"
     --target-dir hdfs://nameservice1//etl/target_path --as-avrodatafile --verbose -m 1
 
 ----
+
+Enabling logical types in parquet import and also turning on padding with 0s, while specifying default precision and scale as well:
+----
+$ sqoop import -Dsqoop.parquet.decimal_padding.enable=true -Dsqoop.avro.logical_types.decimal.enable=true
```

Comment:
I think you wanted to say -Dsqoop.avro.decimal_padding.enable=true and -Dsqoop.parquet.logical_types.decimal.enable=true

Reply:
Yes!